project/BRANCHES-AND-TAGS.md

```diff
@@ -1,32 +1,39 @@
 Branches and tags
 =================
 
-Note: details of the release process for the Engine are documented in the
-[PATCH-RELEASES.md](https://github.com/moby/moby/blob/master/project/PATCH-RELEASES.md).
+> Note: details of the release process of the project are documented in [PATCH-RELEASES.md](PATCH-RELEASES.md).
 
 # Branches
 
-The docker/docker repository should normally have only three living branches at all time, including
-the regular `master` branch:
+`master` serves as the development branch for future releases of the project.
+All changes should be made to the `master` branch, and changes to release branches should only be made in the form of cherry-picked commits, if possible.
+The sponsoring maintainers of a release branch serve as the primary point of contact, and are available to provide guidance on contributing changes to their respective branches.
 
-## `docs` branch
+Keep in mind that release branches only accept bug and security fixes; new features will generally not be considered for backport to release branches.
 
-The `docs` branch supports documentation updates between product releases. This branch allow us to
-decouple documentation releases from product releases.
+Currently (and previously) maintained release branches are documented in the table below:
 
-## `release` branch
+| Branch Name                                    | Sponsoring Maintainer(s)                       | Contribution Status   | Expected End of Maintenance  | Known Distributors                                                   |
+|------------------------------------------------|------------------------------------------------|-----------------------|------------------------------|----------------------------------------------------------------------|
+| master (development branch)                    | The Moby Project [MAINTAINERS](../MAINTAINERS) | N/A                   | -                            | N/A                                                                  |
+| 24.0                                           | @thaJeztah, @rumpl, @neersighted               | Maintained            | TBD                          | [Docker, Inc.](https://docker.com)                                   |
```

Comment:
```suggestion
| 24.0                                           | [@thaJeztah](https://github.com/thaJeztah), @rumpl, @neersighted               | Maintained            | TBD                          | [Docker, Inc.](https://docker.com)                                   |
```

For clickability and for clarifying that these are not Twitter (X) IDs

Reply:
I was for some reason expecting GitHub to linkify these; I agree we should link them to the GitHub profiles.

Reply:
Hello, 
are there any informations about the versions beyond 24.0? Is 24 still maintained?

Reply:
24.0 is not maintained; currently maintained branches;

- `master` / `main`
- `27.0` (which will be renamed to `27.x`) - current release (v27.x.x)
- `25.0` (will be used as base for the next LTS version of Mirantis Container Runtime) 
- `23.0` (branch used as base for current LTS version of Mirantis Container Runtime)

With the disclaimer that older branches depend on versions of the BuildKit dependency that are no longer maintained (the BuildKit project currently doesn't provide maintained long-lived branches)

Reply:
Thank you very much :)

Reply:
I should also add that the 25.0 branch is not _yet_ used by Mirantis; normally, we consider branches with no "owner" (no maintainer involved in maintaining the branch) would be considered "frozen", however Mirantis maintainers have indicated their intent to use that branch for their next LTS release.

@austinvazquez (working at AWS) is not (yet) a maintainer, but requested some backports, and with the knowledge that Mirantis (who does provide maintainers) has the commitment to use that branch in future, we accepted those backports.

Reply:
Is all of this information hidden in the documentation or somewhere else? I searched about 1h before asking here with many different answers but non from docker or moby